.github/workflows/build-and-test.yaml

```diff
@@ -88,7 +89,7 @@ jobs:
           cd tests/third_party/yosys
           make config-gcc
           echo "ENABLE_ABC := 0" >> Makefile.conf
-          make -j6
+          make -j$(nproc)
```

Comment:
I think I overrode nprocs because of OOMs. Maybe conditions changed and we don't need the override, we'll see